rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -238,8 +425,6 @@ void COCOMetaDataReader::read_all(const std::string &path)
                         parser.SkipValue();
                     }
                 }
-                _label_info.insert(std::make_pair(id, continuous_idx));
```

Comment:
This will break mask training and possibly future SSD trainings. Please add this back and try to transform the pixelwise mask labels after this if its needed